l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -183,6 +183,19 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         require(token != address(0), "BH: token not set");
         baseTokenAssetId[_chainId] = DataEncoding.encodeNTVAssetId(block.chainid, token);
     }
+    
+    /// @notice Used to set the legacy chain address for the upgrade.
+    /// @notice This has to be used after the BH but before the STM is upgraded.
```

Comment:
I think the comment is not correct, it has to be used after STM is upgraded, since `getHyperchainLegacy` is available only on the new one